kubernetes/run-unit-tests.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -e
+
+mkdir "/srv/http/concent_api/concent_api/settings/config_map"
+touch "/srv/http/concent_api/concent_api/settings/config_map/local_settings.py"
+
+/usr/local/bin/concent-api-manage.sh                    \
+    "test"                                              \
+    "--failfast"                                        \
+    "--settings=concent_api.settings.docker_testing"
```

Comment:
You don't really have to put these options in quotes.

Reply:
Done.